mgmt_cli_test.py

```diff
@@ -175,22 +200,14 @@ def generate_load_and_wait_for_results(self):
         self.log.info(f'load={load_results}')
 
     # pylint: disable=too-many-arguments
-    def verify_backup_success(self, bucket, cluster_id, keyspace_name='keyspace1', table_name='standard1',
-                              local='/tmp/backup', truncate=True):
-        # FIXME: it will only work with 1 single bucket
-        self.download_file_from_backup_repo(local=local, bucket=bucket, cluster_id=cluster_id)
-        node = self.db_cluster.nodes[0]
-        path_to_tmp = f'{local}/{keyspace_name}/table/{table_name}'
-        table_id = node.remoter.run(f'ls {path_to_tmp}').stdout.strip()
-        path_to_upload = f'/var/lib/scylla/data/{keyspace_name}/{table_name}-{table_id}/upload'
-        node.remoter.run(f'sudo cp {path_to_tmp}/{table_id}/* {path_to_upload}/.')
-        node.remoter.run(f'sudo chown scylla:scylla -Rf {path_to_upload}')
+    def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspace1', tables_names='standard1',
+                              truncate=True):
+        per_keyspace_tables_dict = {keyspace_name: [tables_names]}
```

Comment:
Can `tables_names` even contain multiple tables? If not, it should be renamed to `table_name`. And in this case, can the verification function only handle one table at a time? Shouldn't we be prepared to run it for multiple tables?

Reply:
i have checked (and i'm running a counter proof) that it must now work with a list, hence the name stays ```tables_names```.
have done a minor change to have it supported.